core/controllers/cron.py

```diff
@@ -298,3 +299,31 @@ def get(self) -> None:
             job_class=(
                 suggestion_stats_computation_jobs
                 .GenerateTranslationContributionStatsJob))
+
+
+class CronMailChapterPublicationsNotificationsHandler(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+):
+    """Handler for mailing curriculum admins to remind them about behind
+    schedule and upcoming (within 14 days) chapter launches.
```

Comment:
Is the `14` days period fixed or can be changed in the future? If it can be changed then I would suggest creating a constant variable in the constants.ts file.

Reply:
Done